src/main/java/core/basesyntax/resources/input.csv

```diff
@@ -0,0 +1,8 @@
+b,banana,20
```

Comment:
[Place the input and output files into the `src/main/resources` folder.](https://mate-academy.github.io/jv-program-common-mistakes/java-core/solid/fruit-shop#place-the-input-and-output-files-into-the-srcmainresources-folder)